src/main/java/seedu/address/model/property/Buyer.java

```diff
@@ -0,0 +1,83 @@
+package seedu.address.model.property;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Objects;
+
+import seedu.address.model.field.Email;
+import seedu.address.model.field.Name;
+import seedu.address.model.field.Person;
+import seedu.address.model.field.Phone;
+import seedu.address.model.field.Price;
+
+public class Buyer extends Person {
+
+    private final Price max;
```

Comment:
I think it might better if this is renamed to maxPrice or priceMax.
`max` may be short, however, when it is used elsewhere, e.g. in JsonAdapted classes, calling this `String max` would be confusing. And if I call it `String price` and map the json field `price` to the domain object field `max`, this might be confusing as well.

Reply:
I have been thinking about either using `budget` or `maxPrice` (mostly cos `budget` is one word), but I'll set it to `maxPrice` for now